src/pacer.js

```diff
@@ -288,6 +300,24 @@ let PACER = {
     }
   },
 
+  // Returns the case ID for a queries list page.
+  getCaseIdFromIQuerySummary: function(){
+    let tableContainer = document.querySelector("#cmecfMainContent table");
+    let anchors = tableContainer.getElementsByTagName("a");
+    // End this function early if there are no anchor tags
+    if (!anchors.length){ return null; }
+    let lastAnchor = anchors[anchors.length-1]
+    // the next secuence of instruccions will match the digits at the end of the href. 
+    // The href attribute of the anchor tags in the table has the following format:
+    //
+    //    https://ecf.mied.uscourts.gov/cgi-bin/qryDocument.pl?360406
+    //
+    // We can grab the pacer case ID from this url if we get all the digits after the ?.
```

Comment:
Check your spelling here, please. I recommend turning on spellchecker in English if you don't have it on in your editor already.